generators/newfpmpage/index.js

```diff
@@ -0,0 +1,117 @@
+const path = require("path");
+const Generator = require("yeoman-generator");
+const fpmWriter = require("@sap-ux/fe-fpm-writer");
+const serviceWriter = require("@sap-ux/odata-service-writer");
+const axios = require("@sap-ux/axios-extension");
+
+module.exports = class extends Generator {
+    static displayName = "Enables the Fiori elements flexible program model";
+
+    async prompting() {
+        const modules = this.config.get("uimodules") || [];
+        
+        this.answers = await this.prompt([
+            {
+                type: "list",
+                name: "moduleName",
+                message: "To which module do you want add a custom FPM page?",
+                choices: modules,
+                when: !!modules && modules.length > 1
```

Comment:
same like for the other generator - modules can't be undefined

Reply:
I created a utils function and reused in the two places I need it as well as where I copied it from https://github.com/ui5-community/generator-ui5-project/pull/42/commits/adc10cdf027189d05c772d521167466b01ee63d1